system/CLI/BaseCommand.php

```diff
@@ -96,7 +96,7 @@ public function __construct(LoggerInterface $logger, Commands $commands)
     /**
      * Actually execute a command.
      *
-     * @param array<string, mixed> $params
+     * @param array<int|string, string> $params
```

Comment:
How about if the option is used as a flag (i.e. without values)? `php spark foo --sort`
I think the value would be `true` or `null`? Not sure which one.

Reply:
`null`.

Reply:
Fixed.

Reply:
Ah yes. The `true` is when you use `CLI::getOption()` to get the value.